contrib/opencensus-ext-azure/README.rst

```diff
@@ -89,6 +89,29 @@ WARNING: For this feature to work, you need to pass a dictionary to the custom_d
     properties = {'custom_dimensions': {'key_1': 'value_1', 'key_2': 'value_2'}}
     logger.warning('action', extra=properties)
 
+You can pass a callback function to the exporter to process telemetry before it is exported. Your callback function must
+accept an [envelope](https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/Envelope.bond)
```

Comment:
Adding references to bond files could confuse users big time here unless you do some bond processing in this project already, having an AzureEnvelope class or something like that would be way cleaner for external and internal code